src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -1963,18 +1963,10 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Threading\LowLevelLifoSemaphore.Windows.cs" Condition="'$(TargetsWindows)' == 'true'" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Threading\RegisteredWaitHandle.Portable.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(FeatureObjcWrappers)' == 'true'">
+  <ItemGroup Condition="'$(FeatureObjcBridge)' == 'true'">
```

Comment:
I think we should agree on the casing of "ObjC" I prefer "ObjC" rather than "Objc" - thoughts?

Reply:
I don't particularly care which casing we use.

Reply:
`ObjC` sounds good to me.